lp-steam/templates/csgo/server.j2

```diff
@@ -42,4 +62,27 @@ sv_minupdaterate "{{ item.value.tickrate | default('128') }}"
 sv_minrate 80000
 sv_maxrate 0
 
-exec rules_{{ item.value.rules | default('esl5on5') }}.cfg
+{% if item.value.backup | default(false) %}
+mp_backup_round_auto 1 
+mp_backup_round_file backup 
+mp_backup_round_file_pattern {{ csgo_backup_round_file_pattern | default('%prefix%_%date%_%time%_%map%_round%round%.txt') }}
+{% endif %}
+
+{% if item.value.overtime | default(false) %}
+mp_overtime_enable 1
+mp_overtime_maxrounds 6
+mp_overtime_startmoney 10000
+{% endif %}
+
+sv_hibernate_when_empty 0
+sv_hibernate_postgame_delay 180
+
+sv_pure 1
```

Comment:
`sv_pure` and `sv_pure_kick_clients` are set to 1 in esl5on5; but I would echo them here. Are you sure `sv_pure_trace` is supposed to be on?